system-contracts/contracts/Constants.sol

```diff
@@ -193,3 +202,15 @@ uint8 constant ERA_VM_BYTECODE_FLAG = 1;
 uint8 constant EVM_BYTECODE_FLAG = 2;
 
 address constant SERVICE_CALL_PSEUDO_CALLER = 0xFFfFfFffFFfffFFfFFfFFFFFffFFFffffFfFFFfF;
+
+/// @dev Pubdata commitment scheme used for DA.
+/// @param NONE Invalid option.
+/// @param EMPTY_NO_DA No DA commitment, used by Validiums.
+/// @param PUBDATA_KECCAK256 Keccak of stateDiffHash and keccak(pubdata). Can be used by custom DA solutions.
+/// @param BLOBS_AND_PUBDATA_KECCAK256 This commitment includes EIP-4844 blobs data. Used by default RollupL1DAValidator.
+enum L2DACommitmentScheme {
+    NONE,
+    EMPTY_NO_DA,
+    PUBDATA_KECCAK256,
+    BLOBS_AND_PUBDATA_KECCAK256
```

Comment:
What will happen to AVAIL_l2_da_validator?

Reply:
PUBDATA_KECCAK256